blueprints/reducer/files/__root__/reducers/__name__.js

```diff
@@ -0,0 +1,38 @@
+// __name__ REDUCER
```

Comment:
mistakenly I had thought ember-cli would populate with the name but since it doesn't I've removed